lib/fizzy/parser_expr.cpp

```diff
@@ -147,6 +147,8 @@ parser_result<Code> parse_expr(
     const auto function_arity = static_cast<uint8_t>(module.typesec[func_type_idx].outputs.size());
     // The function's implicit block.
     control_stack.emplace(Instr::block, function_arity, 0);
+    
+    local_count += module.typesec[func_type_idx].inputs.size();
```

Comment:
I guess the suggestion here is to keep `local_count` const and have another local variable here.